### ISSUE TITLE
Updated Survey Header and Footer with Crisis Language

### ISSUE THREAD

**Issue by clantosswett** [department-of-veterans-affairs/va.gov-team#57631]
## Issue Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

The VA.gov surveys still do not have updated header language to include crisis information. 

---
## Tasks
- [x] Receive feedback/approval from sitewide content team for new language and formatting.
- [x] Receive approval from VSignal leaders
- [x] Make updates to survey and push to production

## Acceptance Criteria
- [ ] Content team and VSignals team approve new header format and content.
- [ ] Updated content and formatting live in production surveys.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by clantosswett** [comment]
[The issue currently is with the content team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52172). Awaiting feedback on the suggested changes. from them. Will follow up with them next week.

**Comment by clantosswett** [comment]
Received feedback from the content team. Will be sending that feedback back over to the Visgnals team and hopefully will finalize and make a plan for completing any customization needs and getting changes live in production.

**Comment by aubreyarcangel** [comment]
I created [a ticket](https://app.zenhub.com/workspaces/contact-center-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/57780) to track header styling

**Comment by clantosswett** [comment]
Waiting for a final mock up from Sarkis that can be reviewed by the content and VSignals team for final approval.

**Comment by ATMiddleton** [comment]
@aubreyarcangel will add ARIA labels.  Edits are in staging status.  We’ve made changes to the A11 VHA survey in our staging environment so that the team can review the changes. You can review the changes we’ve made by going to the link below:
https://staging.va.gov/health-care/eligibility/“

**Comment by aubreyarcangel** [comment]
[Here's the ticket](https://app.zenhub.com/workspaces/contact-center-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/58840) tracking the aria-labels and additional elements

We have a new blocker. I've identified that the Cross-Origin Resource Sharing (CORS) policy is currently blocking the implementation of adding aria-labels to the Medallia header. Here's an explanation of why CORS is causing issues and some possible solutions we can explore:

Understanding CORS:
CORS is a security mechanism web browsers implement to restrict cross-origin HTTP requests. It ensures that resources (e.g., scripts, fonts, or APIs) requested from one domain are only accessible to requests originating from the same domain unless explicitly allowed. This policy prevents potential security vulnerabilities that could arise from unauthorized access to sensitive data. Here's a. more in-depth documentation on CORS.

To overcome this CORS-related blocker, I have contacted the VSignals team to discuss possible solutions.
